src/runtime/linux.rs

```diff
@@ -363,6 +363,7 @@ pub struct LinuxCpu {
 /// LinuxPids for Linux cgroup 'pids' resource management (Linux 4.3).
 pub struct LinuxPids {
     #[serde(default)]
+    #[getset(get = "pub")]
```

Comment:
Try replacing Getters with CopyGetters in line 355 instead.
```suggestion
 #[derive(Builder, Clone, Copy, Debug, Default, Deserialize, Eq, CopyGetters, PartialEq, Serialize)]
```

Reply:
All right. I changed it that way.

Reply:
With this change, there are many places that return the value reference, such as cgroup crate in youki, and you may get the following error.
In the case of `#[getset(get = "pub")]`, it worked.

```
error[E0515]: cannot return value referencing temporary value
  --> src/v1/pids.rs:32:20
   |
31 |         if let Some(pids) = &controller_opt.resources.pids() {
   |                              ------------------------------- temporary value created here
32 |             return Some(pids);
   |                    ^^^^^^^^^^ returns a value referencing data owned by the current function
```

Reply:
Maybe I can solve it myself. Thanks.